app/importers/daily_calls/twilio.rb

```diff
@@ -0,0 +1,15 @@
+module DailyCalls
+  module Twilio
+    autoload :CallRecord, 'daily_calls/twilio/call_record'
+    autoload :Retriever, 'daily_calls/twilio/retriever'
+    autoload :Saver, 'daily_calls/twilio/saver'
+
+    module_function
```

Comment:
I'm not a huge fan of this `module_function` style of single-use 'objects'. Is there any reason why we can't use `call`able style objects that have their dependencies injected instead? I think this style lends itself to change and ensures the dependencies for `import` are not opaque - as they are now.

This applies here and elsewhere we've used `module_function`.